### PR TITLE
LPS-147860 Add test ResponsiveModeClearButtonShouldDisplayTooltip

### PR DESCRIPTION
**Pull request by susanchen3**
**Background**
Create automated tests for Management Toolbar.

**Test Plan**
Given a user of the portal on responsive mode
When the user opens any section with the management toolbar
And hovers on the 'clear' button
Then a tooltip message will be displayed

**JIRA Ticket:**
https://issues.liferay.com/browse/LPS-147860

### CODE REVIEW

**Comment by susanchen3** [comment]
ci:test:sf

**Comment by susanchen3** [comment]
ci:test:frontend:nocompile

File: portal-web/test/functional/com/liferay/portalweb/tests/enduser/frontendinfrastructure/uiinfrastructure/wysiwyg/ManagementToolbar.testcase [hunk 1]
```diff
@@ -319,6 +319,26 @@ definition {
...
 	}
 
+	@description = "LPS-147860. Assert tooltip message will be displayed when hovered over the clear button in responsive mode"
+	@priority = "3"
+	test ResponsiveModeClearButtonShouldDisplayTooltip {
+		task ("Given a window size set to phone size") {
+			SetWindowSize(value1 = "360,720");
+		}
+
+		task ("And hovers over the clear button") {
```
Comment (john-co) [inline portal-web/test/functional/com/liferay/portalweb/tests/enduser/frontendinfrastructure/uiinfrastructure/wysiwyg/ManagementToolbar.testcase:329]: When mouse hover over the clear button

**Review by john-co — APPROVED** [review]
Looks good to me, please include the small adjustment and forward.

**Comment by susanchen3** [comment]
ci:forward

**Pull request closed without merging**